source/frontend/src/features/mapSideBar/shared/sidebarPathGenerator.ts

```diff
@@ -174,7 +174,7 @@ const usePathGenerator: IPathGenerator = () => {
     editDetails,
     addDetail,
     editProperties,
-    showFilePropertyIndex,
+    showFilePropertyId: showFilePropertyIndex,
```

Comment:
nit: I would consider renaming the internal function as well to match the new name (also the parameters in the existing function say "menuIndex" - where in new code it would be filePropertyId